src/functions.php

```diff
@@ -0,0 +1,26 @@
+<?php
```

Comment:
Perhaps we can even drop these functions (what are their real value ?)

Else, composer.json need to be aware of them
```
    "autoload": {
        "psr-4": {
            "Fedora\\Autoloader\\": "src/"
        },
        "files": ["src/functions.php"]
    },
```

Or even simpler, don't use composer autoloader, but only our
```
    "autoload": {
        "files": ["src/autoload.php"]
    },

```